lib/chef_apply/target_host/aix.rb

```diff
@@ -0,0 +1,55 @@
+module ChefApply
+  class TargetHost
+    module Aix
+
+      def omnibus_manifest_path
+        # Note that we can't use File::Join, because that will render for the
+        # CURRENT platform - not the platform of the target.
+        "/opt/chef/version-manifest.json"
+      end
+
+      def mkdir(path)
+        run_command!("mkdir -p #{path}")
+      end
+
+      def chown(path, owner)
+        # owner ||= user
```

Comment:
In aix platform chown is failing as path is `""`. But in case of other's platform it is something like `tmp/chef_J9Se`. For now chown is commented out.